### ISSUE TITLE
errored output in RFC header/footer

### ISSUE THREAD

**Issue by ietf-svn-bot** [ietf-tools/xml2rfc#34]
`owner:jbothun@concentricsky.com` `resolution_fixed` `type_defect`   |    by ahagens@amsl.com

___

Regarding the running header/footer of an RFC.
Output shown below is nroff output unless noted otherwise.

1) center footer is "std" but should be "Standards Track",
"bcp" but should be "Best Current Practice", etc.

v1 output (desired):
.ds CF Best Current Practice

v2 output:
.ds CF bcp

2) right footer is "FORMFEED[Page] X"
but should be "[Page X]

v1 output (desired):
.ds RF FORMFEED[Page %]

v2 output:
.ds RF FORMFEED[Page] %

3) left header is the RFC number with out 'RFC'.

desired output (v1 output has an extraneous \0):
.ds LH RFC 6365

v2 output:
.ds LH 6365

In summary:

This is the text output when the v1 NROFF output is fed to nroff -ms:

```
Hoffman & Klensin         Best Current Practice                 [Page 1]
^L
RFC 6365            Internationalization Terminology      September 2011
```

This is the text output when the v2 NROFF is fed to nroff -ms:

```
Hoffman & Klensin                  bcp                 FORMFEED[Page] 1





6365                Internationalization Terminology      September 2011
```

___
_Issue migrated from trac:34 at 2022-02-05 12:23:55 +0000_
